notebooks/community/structured_data/rapid_prototyping_bqml_automl.ipynb

```diff
@@ -0,0 +1,1527 @@
+{
```

Comment:
<p>Add cleanup steps for dataset, model, pipeline, etc. </p><p> </p><p>See other notebooks for examples: https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/master/notebooks/official/pipelines</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>